app/api/v2/views/auth.py

```diff
@@ -70,14 +70,42 @@ def login_user():
         userfound = '{}'.format(auth.get("user", national_id=national_id))
         if userfound != "None":
             response = auth.login_user(national_id, password)
-            
-            return sp.sdict(response)
+
```

Comment:
It would be wise if you add a single endpoint for the whole reset password process and add the subsequent checks inside that endpoint.